tests/foreman/maintain/test_service.py

```diff
@@ -149,12 +149,18 @@ def test_positive_service_enable_disable(sat_maintain):
 
     :customerscenario: true
     """
+    result = sat_maintain.cli.Service.stop()
+    assert 'FAIL' not in result.stdout
+    assert result.status == 0
     result = sat_maintain.cli.Service.disable()
     assert 'FAIL' not in result.stdout
     assert result.status == 0
     result = sat_maintain.cli.Service.enable()
     assert 'FAIL' not in result.stdout
     assert result.status == 0
+    sat_maintain.power_control(state='reboot')
+    result = sat_maintain.execute('systemctl status foreman.service')
+    assert 'inactive (dead)' not in result.stdout
```

Comment:
We could also use cli.Service.status() here for checking status of foreman.service

Reply:
you need to use `foreman-maintain service status -b --only foreman.service` which runs the same systemd command eventually